custom/icds_reports/management/commands/sql_scripts/fix_past_data_part_1.sql

```diff
@@ -0,0 +1,33 @@
+UPDATE child_health_monthly child_health
+   SET
+      height_measured_in_month = CASE
+        WHEN (gm.height_child_last_recorded >= '%(start_date)s' AND gm.height_child_last_recorded <'%(end_date)s') AND (valid_in_month=1 AND age_tranche::Integer <= 60) THEN 1
+        ELSE
+          0
+        END,
+      current_month_stunting = CASE
+        WHEN NOT (valid_in_month=1 AND age_tranche::Integer <= 60) THEN NULL
+        ELSE
+            current_month_stunting
+        END,
+      stunting_last_recorded = CASE
+        WHEN NOT (valid_in_month=1 AND age_tranche::Integer <= 60) THEN NULL
+        ELSE
+           stunting_last_recorded
+        END,
+      wasting_last_recorded = CASE
+        WHEN NOT (valid_in_month=1 AND age_tranche::Integer <= 60) THEN NULL
+       ELSE
+          wasting_last_recorded
+       END,
+      current_month_wasting = CASE
+        WHEN NOT (valid_in_month=1 AND age_tranche::Integer <= 60) THEN NULL
+        ELSE
+          current_month_wasting
+        END
+   FROM icds_dashboard_growth_monitoring_forms gm
```

Comment:
This wont work for most of the months (same comment i made earlier)

Reply:
Added the script to build the past gm forms